### ISSUE TITLE
Size mismatch between original file and the extracted files - German Wikipedia

### ISSUE THREAD

**Issue by maggieezzat** [attardi/wikiextractor#171]
I used the wikiextractor to extract the german wikipedia. The whole xml file was 20 GB, however the extracted files were around 5 GB. I don't know how that happened, the extraction ran to the end, without any errors. My extracted folders are AA...Az, BA...BZ, CA,CB and CC only. is that okay?

**Comment by HjalmarrSv** [comment]
Yes, that is expected. I have the same experience for other language.

**Comment by PosoSAgapo** [comment]
I have the same confusion about this problem, my 252Mb wiki small dump ends with only 18Mb, did you do a cross-check to valid the output result?

**Comment by HjalmarrSv** [comment]
If you open your dump, you will find that it contains mostly other stuff than the articles. So, expect the output to be a lot smaller than the dump, if you only want the articles. If you want to use all the stuff (discussions, markup, templates, etc) in the dump, then you do not need an extractor. If you want for example discussions you can include that name spaces in the extraction.

**Comment by aadhiraj-code** [comment]
> If you open your dump, you will find that it contains mostly other stuff than the articles. So, expect the output to be a lot smaller than the dump, if you only want the articles. If you want to use all the stuff (discussions, markup, templates, etc) in the dump, then you do not need an extractor. If you want for example discussions you can include that name spaces in the extraction.

I downloaded the whole wikipedia dump (50 gb ish decompressed) without the discussions, but when I extracted it, it only ended up with 500 mb. Is that expected or is that a bug in my code? Just asking because for others it reduced 10x mine a 100x
